.dev/tests/phpunit/includes/test-coblocks-block-assets.php

```diff
@@ -98,7 +98,31 @@ public function test_editor_assets_scripts() {
 		$this->go_to( '/wp-admin/post-new.php' );
 		$this->coblocks_block_assets->editor_assets();
 
-		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered ) );
+		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered,  ) );
```

Comment:
Looks like there is excess spaces before the first closing `)` and maybe missing a parameter?

```suggestion
		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered ) );
```